src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -495,6 +496,10 @@ def __new_substatus_json_for_operation(operation_name, status="Transitioning", c
 
     # region - Status generation
     def __reset_status_file(self):
+        status_file_reset_content = json.dumps(self.__new_basic_status_json())
+        # Create complete status template
+        self.env_layer.file_system.write_with_retry(self.complete_status_file_path, '[{0}]'.format(status_file_reset_content), mode='w+')
+        # Create agent-facing status template
         self.env_layer.file_system.write_with_retry(self.status_file_path, '[{0}]'.format(json.dumps(self.__new_basic_status_json())), mode='w+')
```

Comment:
Why is line 503 not using the variable populated on line 499. 

The purpose of the variable is to avoid the json recreation twice.